docker/px4-dev/Dockerfile_base

```diff
@@ -54,6 +54,13 @@ RUN apt-get update \
 	# cleanup
 	&& rm -rf /var/lib/apt/lists/{apt,dpkg,cache,log} /tmp/* /var/tmp/*
 
+# Add Fast-RTPS
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y --quiet --no-install-recommends install \
+		openjdk-8-jdk-headless \
```

Comment:
Can you merged this into the list above?
Each separate RUN command creates a layer which ends up on docker hub. Packing in together above keeps the image as small as possible (it's pulled every single build).